### PR TITLE
Update description of "Apps are up" to accurately describe job

### PR DESCRIPTION
**Pull request by benvand**
Smoke tests now test 7 apps 

https://github.com/alphagov/digitalmarketplace-functional-tests/blob/master/features/smoke-tests/apps_status.feature
https://trello.com/c/uFfPSPZf/156-change-smoke-tests-description-on-jenkins

### CODE REVIEW

File: job_definitions/smoke_tests.yml [hunk 1]
```diff
@@ -6,13 +6,16 @@
...
       <ul>
         <li>API</li>
         <li>Search API</li>
+        <li>Admin frontend</li>
+        <li>Briefs frontend</li>
+        <li>Brief-responses frontend</li>
         <li>Buyer frontend</li>
         <li>Supplier frontend</li>
-        <li>Admin frontend</li>
+        <li>User frontend</li>
```
Comment (katstevens) [inline job_definitions/smoke_tests.yml]: Should we include the router as an 'app'? The smoke tests certainly would fail without it 😸

Comment (benvand) [reply job_definitions/smoke_tests.yml]: My only problem with this is that if we mention it here we should really mention it on all the functional tests.

Saying that we could add the router to the smoke tests 🤔

Comment (benvand) [reply job_definitions/smoke_tests.yml]: Hmm maybe not though
I don't think we can get the url of a router app

Comment (katstevens) [reply job_definitions/smoke_tests.yml]: OK, makes sense.

**Pull request merged**